airflow/operators/subdag_operator.py

```diff
@@ -37,18 +49,6 @@ def __init__(
             subdag,
             executor=SequentialExecutor(),
             *args, **kwargs):
-        """
-        This runs a sub dag. By convention, a sub dag's dag_id
-        should be prefixed by its parent and a dot. As in `parent.child`.
-
-        :param subdag: the DAG object to run as a subdag of the current DAG.
-        :type subdag: airflow.DAG.
-        :param dag: the parent DAG for the subdag.
-        :type dag: airflow.DAG.
-        :param executor: the executor for this subdag. Default to use SequentialExecutor.
-                         Please find AIRFLOW-74 for more details.
-        :type executor: airflow.executors.
-        """
```

Comment:
Curious. I thought sphinx looked here for class-level docs. I guess not!

Reply:
I think it has something to do with configurations:
- https://stackoverflow.com/questions/5599254/how-to-use-sphinxs-autodoc-to-document-a-classs-init-self-method
- http://www.sphinx-doc.org/en/stable/ext/example_google.html